### ISSUE TITLE
Read uncommitted is impossible under Firebird

### ISSUE THREAD

**Issue by gsbelarus** [hgourvest/node-firebird#147]
There is no way in Firebird that one transaction will see any  UNCOMMITTED changes made by other transactions. So, the name ISOLATION_READ_UNCOMMITTED is incorrect.

```
const
    ISOLATION_READ_UNCOMMITTED          = [isc_tpb_version3, isc_tpb_write, isc_tpb_wait, isc_tpb_read_committed, isc_tpb_rec_version],
    ISOLATION_READ_COMMITED             = [isc_tpb_version3, isc_tpb_write, isc_tpb_wait, isc_tpb_read_committed, isc_tpb_no_rec_version],
```

**Comment by mariuz** [comment]
You're right about the semantics: Firebird has no dirty reads, and `ISOLATION_READ_UNCOMMITTED` actually maps to `[isc_tpb_read_committed, isc_tpb_rec_version, ...]` — i.e. READ COMMITTED (record-version). The constant name is kept for API compatibility (it's what other drivers call the weakest level), but what you get is Firebird's weakest real isolation, never dirty reads. Closing as a naming clarification rather than a bug — the behavior is correct and now documented by this issue.
